desktop/ui/src/main/java/org/datacleaner/windows/ElasticSearchDatastoreDialog.java

```diff
@@ -82,7 +80,6 @@ public ElasticSearchDatastoreDialog(WindowContext windowContext, MutableDatastor
         setSaveButtonEnabled(false);
 
         _clientTypeComboBox = new JComboBox<>(ClientType.values());
-        _clientTypeComboBox.setSelectedItem(DEFAULT_CLIENT_TYPE);
```

Comment:
With the removal of this line the private constant DEFAULT_CLIENT_TYPE is no longer used.